net/https-dns-proxy/files/https_dns_proxy.init

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh /etc/rc.common
+
+. /lib/functions/network.sh
+
+START=80
+
+USE_PROCD=1
+PROG=/usr/sbin/https_dns_proxy
+
+start_instance() {
+	local cfg="$1"
+	local listen_addr listen_port user group
+
+	config_get listen_addr "$cfg" listen_addr
+	config_get listen_port "$cfg" listen_port
+	config_get user "$cfg" user
+	config_get group "$cfg" group
+
+	procd_open_instance
+	procd_set_param command ${PROG} -d \
+		-l "$listen_addr" -p "$listen_port" \
+		-u "$user" -g "$group"
+	procd_set_param respawn
+	procd_open_trigger
+	procd_add_config_trigger "config.change" "https_dns_proxy" /etc/init.d/https_dns_proxy reload
+	procd_close_trigger
+	procd_close_instance
+
+}
+
+service_triggers()
+{
```

Comment:
Putting this curly bracket at the same line as the func name to make style consistent.